### DISCUSSION TITLE
Welcome to ASUS-TUF-F15-OPENCORE Discussions!

### DISCUSSION THREAD

**Discussion by ad1tyx666** [Vincent4440/ASUS-TUF-F15-OPENCORE#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sulthannaf** [comment]
I have a asus tuf f15 fx506lh, now windows 11, is it support? Wifi intel. Any some bug for my laptop...?![image](https://user-images.githubusercontent.com/104815393/200026790-213a964d-d2af-4dec-92e5-543b94246afa.jpeg) ![image](https://user-images.githubusercontent.com/104815393/200027043-d3dc4b04-0e66-47eb-9236-c3d3666a3bd6.jpeg)

**Reply by ad1tyx666** [reply]
Yes your wifi will work with your card you just have to add https://github.com/OpenIntelWireless/itlwm and https://github.com/OpenIntelWireless/IntelBluetoothFirmware kexts in your efi, about bugs it depens on the version of the os you will be using, as far i have tested hackintosh in my fx506lh there's no problem i have faced except wifi and bluetooth (i have an mideatek wifi card that's why), thanks for using my efi and sorry for late reply
if you are new to hackintoshing you could see tutorials on how to add intel wifi and bluetooth support in your efi if you are still facing any problems you can contact me on [aditya.singh2008v@gmail.com](mailto:aditya.singh2008v@gmail.com) i will help you for sure

**Comment by ad1tyx666** [comment]
Yes your wifi will work with your card you just have to add https://github.com/OpenIntelWireless/itlwm and https://github.com/OpenIntelWireless/IntelBluetoothFirmware kexts in your efi, about bugs it depens on the version of the os you will be using, as far i have tested hackintosh in my fx506lh there's no problem i have faced except wifi and bluetooth (i have an mideatek wifi card that's why), thanks for using my efi and sorry for late reply

**Comment by ad1tyx666** [comment]
one more thing if you are new to hackintoshing you could see tutorials on how to add intel wifi and bluetooth support in your efi if you are still facing any problems you can contact me on aditya.singh2008v@gmail.com i will help you for sure
